README.md

```diff
@@ -15,9 +15,10 @@
 ![Released Focal Version](https://img.shields.io/ubuntu/v/ubuntu-advantage-tools/focal?label=Focal&logo=ubuntu&logoColor=white)
 ![Released Jammy Version](https://img.shields.io/ubuntu/v/ubuntu-advantage-tools/jammy?label=Jammy&logo=ubuntu&logoColor=white)
 
-The Ubuntu Advantage (UA) Client provides users with a simple mechanism to
-view, enable, and disable offerings from Canonical on their system. The
-following entitlements are supported:
+The Ubuntu Advantage (UA) Client is the offical plataform to enable Canonical offering on your
```

Comment:
```suggestion
The Ubuntu Advantage (UA) Client is the official platform to enable Canonical offerings on your
```

Reply:
Also "platform" kind of implies something bigger/broader than what we offer I think. Maybe replace "platform" with just "tool"